doc/specs/stdlib_logger.md

```diff
@@ -8,19 +8,20 @@ title: logger
 ## Introduction
 
 This module defines a derived type, its methods, a variable, and
-constants to be used for the reporting of errors and other
-information. The derived type, `logger_type`, is to be used to define
-both global and local logger variables. The `logger_type` methods serve
-to configure the loggers and use the logger variables to report
-messages to a variable specific list of I/O units termed
-`log_units`. The variable, `global_logger`, of type `logger_type`, is
-intended to serve as the default global logger. The constants serve as
+constants to be used for the reporting of errors, for filtering the 
+log messages, and other information. The derived type, `logger_type`,
```

Comment:
This comment might be off due to me not being a native English speaker. I am not sure if "filtering" is the most suitable word for what you are trying to describe. I would be happy to leave the concept of levels out in the first paragraph. In that case the sentence could finish as `"...to be used for the reporting of errors, displaying messages, and other information."`

Reply:
"filtering" was used in the docs of both Julia and Python logging. I would be happy to change it to a more suitable word.
Anyway, I will change the sentence as you proposed.

Reply:
Changed as proposed. The word `filtering` has been removed.